packages/api/Dockerfile

```diff
@@ -1,17 +1,20 @@
 # 이미지 full
 FROM node:latest
 
+RUN npm install -g sequelize-cli
+
 #/app 디렉토리 생성
 RUN mkdir -p /api
 # /app 디렉토리를 WORKDIR로 설정
 WORKDIR /api
+
 # 현재 Dockerfile 있는 경로의 모든 파일을 /app 에 복사
 ADD . /api
 # npm install 을 실행
 RUN npm install
 
 # 환경변수 NODE_ENV의 값을 development로 설정
-ENV NODE_ENV development
+ENV NODE_ENV "production"
```

Comment:
The value of NODE_ENV doesn't match with its comment. Please make they have same values.

Reply:
Fixed the comment